imagify.php

```diff
@@ -3,7 +3,7 @@
  * Plugin Name: Imagify
  * Plugin URI: https://wordpress.org/plugins/imagify/
  * Description: Dramaticaly reduce image file sizes without losing quality, make your website load faster, boost your SEO and save money on your bandwidth using Imagify, the new most advanced image optimization tool.
- * Version: 1.9.10
+ * Version: 1.9.11
```

Comment:
Please add the following header fields:

- Requires at least: The lowest WordPress version that the plugin will work on.
- Requires PHP: The minimum required PHP version.

[See WordPress Plugin manual for more info](https://developer.wordpress.org/plugins/plugin-basics/header-requirements/#header-fields).